bitrise.yml

```diff
@@ -1,54 +1,116 @@
-format_version: "11"
+---
+format_version: "17"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: other
 
-app:
-  envs:
-  - BIN_NAME: bitrise
+trigger_map:
+- push_branch: master
+  pipeline: ci
+- pull_request_source_branch: "*"
+  pipeline: ci
+- tag: "*.*.*"
+  workflow: release
+  enabled: false
+
+pipelines:
+  ci:
+    stages:
+    - run_tests: { }
+
+stages:
+  run_tests:
+    workflows:
+    - run_lint: { }
+    - run_unit_tests_macos: { }
+    - run_unit_tests_linux: { }
+    - run_integration_tests_macos: { }
+    - run_integration_tests_linux: { }
+    - run_docker_integration_tests_linux: { }
+    - test_binary_build_macos: { }
+    - test_binary_build_linux: { }
 
 workflows:
-  check:
+  run_lint:
     steps:
-    - git::https://github.com/bitrise-steplib/steps-check.git:
-        inputs:
-        - workflow: lint
-        - skip_step_yml_validation: "yes"
-    - go-list: { }
-    - script:
-        title: Go test
-        inputs:
-        - content: go test -p 1 $BITRISE_GO_PACKAGES
+    - bundle::run_lint: { }
 
-  test:
-    title: Runs tests
-    after_run:
-    - check
-    - integration-test
+  run_unit_tests_macos:
+    steps:
+    - bundle::setup_go_junit_report: { }
+    - bundle::run_unit_tests: { }
+    meta:
+      bitrise.io:
+        stack: osx-xcode-16.0.x
+        machine_type_id: g2.mac.large
 
-  integration-test:
-    title: Runs integration tests
+  run_unit_tests_linux:
     steps:
-    - script:
-        title: Run integration tests
-        inputs:
-        - content: |-
-            #!/usr/bin/env bash
-            set -ex
+    - bundle::setup_go_junit_report: { }
+    - bundle::run_unit_tests: { }
+    meta:
+      bitrise.io:
+        machine_type_id: elite-xl
+        stack: linux-docker-android-22.04
 
-            current_bitrise="$(pwd)/_tmp/test_bitrise"
-            export CGO_ENABLED=0
-            go build -o "$current_bitrise"
+  run_integration_tests_macos:
+    steps:
+    - bundle::setup_go_junit_report: { }
+    - bundle::run_integration_tests: { }
+    meta:
+      bitrise.io:
+        stack: osx-xcode-16.0.x
```

Comment:
How about running on `osx-xcode-latest`, which is a pointer to the latest edge stack? We do this in some build platform projects, it has two benefits:
1. Dogfoods the edge stacks, which often have experimental fixes and changes applied
2. This stack ID will never be removed or become invalid, there is no need to manually update it after a stack gets deprecated

Reply:
Fixed in https://github.com/bitrise-io/bitrise/pull/1005/commits/945ccd87391100cd63f2dedd5a6720dd3030794a